SETUP/Makefile

```diff
@@ -22,7 +22,7 @@ lint_code:
 	$(SELF_DIR)lint_php_files.sh $(SELF_DIR)..
 	$(SELF_DIR)lint_json_files.sh $(SELF_DIR)..
 
-lint: lint_charsuites lint_less lint_code
```

Comment:
lint_less is not a target so make complained when I ran ```make lint```. The github automation call the targets individually so this doesn't show up in checks.

Reply:
Oops. This should be `lint_css` instead though.